PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -238,10 +240,26 @@ def process(self, event):
             else:
                 photons = [ g for g in event.selectedPhotons ] 
 
-        self.gamma_cleanJetsAll = cleanNearestJetOnly(self.cleanJetsAll, photons, self.jetGammaDR)
+        self.gamma_cleanJetaAll = []
+        self.gamma_noIdCleanJetsAll = []
+
+        if getattr(self.cfg_ana, 'cleanFromLepAndGammaSimultaneously', False):
+            if hasattr(self.cfg_ana, 'jetGammaLepDR'):
+                self.jetGammaLepDR =  self.cfg_ana.jetGammaLepDR 
+            elif (self.jetGammaDR == self.jetLepDR):
+                self.jetGammaLepDR = jetGammaDR
+            else:
+                print "WARNING: dR for simultaneous cleaning of jets from leptons and photons is not defined, and dR(gamma, jet)!=dR(lep, jet). Using default (0.4)"
```

Comment:
Last small requests before merging this PR: 
* add the `cleanFromLepAndGammaSimultaneously` to the default configuration at the bottom
* move this part of the check, that happens at configuration time, to the constructor of the analyzer and change the `print message` with `raise RuntimeError, message`